src/bundle/Templating/Twig/ComponentExtension.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\Templating\Twig;
+
+use EzSystems\EzPlatformAdminUi\Component\Registry as ComponentRegistry;
+use EzSystems\EzPlatformAdminUi\Component\Renderable;
+use EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException;
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+class ComponentExtension extends Twig_Extension
+{
+    protected $registry;
+
+    public function __construct(
+        ComponentRegistry $registry
+    ) {
+        $this->registry = $registry;
+    }
+
+    public function getFunctions()
+    {
+        return [
+            new Twig_SimpleFunction(
+                'ezplatform_adminui_component_group',
```

Comment:
I believe it would be more streamlined with other parts of Admin UI if the name was `ezplatform_admin_ui_component_group`